ui/revenuecatui/src/test/kotlin/com/revenuecat/purchases/ui/revenuecatui/extensions/Offering.kt

```diff
@@ -0,0 +1,22 @@
+package com.revenuecat.purchases.ui.revenuecatui.extensions
+
+import com.revenuecat.purchases.Offering
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.paywalls.PaywallData
+
+@Suppress("LongParameterList")
+internal fun Offering.copy(
```

Comment:
Hmm I see we have some copy methods in the `purchases` module, with a `@InternalRevenueCat` annotation... Would it be worth doing the same for these ones? Not sure if they might be useful there at some point... In any case, not a blocker, I'm also ok with this.